src/Config/Environment.php

```diff
@@ -151,4 +153,35 @@ public function isMasterBranch(): bool
         return !empty($branchName)
             && preg_match(self::GIT_MASTER_BRANCH_RE, $branchName);
     }
+
+    /**
+     * Checks whether application has specific mount.
+     *
+     * @param string $name
+     * @return bool
+     */
+    public function hasMount(string $name): bool
+    {
+        $application = $this->getApplication();
+
+        /**
+         * File config does not have slash in the beginning.
+         */
+        $name = ltrim($name, '/');
+
+        if (isset($application['mounts'][$name])) {
+            return true;
+        }
+
+        /**
+         * Environment variable have slash in the beginning.
+         */
+        $name = '/' . $name;
+
+        if (isset($application['mounts'][$name])) {
+            return true;
+        }
+
+        return false;
```

Comment:
Seems these lines can be replaced with:
```
        return isset($application['mounts'][$name]) || isset($application['mounts']['/' . $name])
```

Reply:
It's a generally a bad idea to perform operations inside of array index, moved to variables

Reply:
I don't see any danger in this operation of concatenation.
The code will become more readable if you change 13 rows with one.
Also, you can change to `array_key_exists` to avoid operations in the array index.
```php
    return array_key_exists($name, $application['mounts']) || array_key_exists('/' . $name, $application['mounts']);
```

Reply:
Fixed